core/sdk-core/src/main/java/software/amazon/awssdk/core/RequestOverrideConfiguration.java

```diff
@@ -494,11 +496,13 @@ default B putRawQueryParameter(String name, String value) {
          *
          * @param plugin The plugin to add.
          */
+        @SdkPreviewApi
```

Comment:
looks like we missed adding it to `B plugins(List<SdkPlugin> plugins);` above.

Reply:
Good catch, I opened #4580 to fix this and other issues. Thanks!